development.ini.example

```diff
@@ -282,6 +282,16 @@ acl_system = dummy
 ##
 pkgdb_url = https://admin.fedoraproject.org/pkgdb
 
+##
+## Pagure
+##
+pagure_url = https://src.fedoraproject.org/pagure/
+
+##
+## Product Definition Center (PDC)
+##
+pdc_url = https://pdc.fedoraproject.org/
```

Comment:
It could be nice to comment both of these settings and to make sure that these values are the defaults if the settings aren't set.